rfc9114.md

```diff
@@ -59,11 +42,97 @@ normative:
           org: Facebook
           role: editor
 
+  HTTP:
+    title: "HTTP Semantics"
+    date: {DATE}
```

Comment:
It's probably not a good idea to automatically generate the date like this.  Try "1970-01-01" instead.